pandas/tseries/offsets.py

```diff
@@ -971,21 +971,25 @@ class BusinessMonthBegin(MonthOffset):
 
 class _CustomBusinessMonth(_CustomMixin, BusinessMixin, MonthOffset):
     """
-    DateOffset subclass representing one custom business month, incrementing
-    between [BEGIN/END] of month dates.
+    DateOffset subclass representing custom business month(s).
+
+    Increments between %(bound)s of month dates.
 
     Parameters
     ----------
     n : int, default 1
+        The number of months represented.
     normalize : bool, default False
-        Normalize start/end dates to midnight before generating date range
+        Normalize start/end dates to midnight before generating date range.
     weekmask : str, Default 'Mon Tue Wed Thu Fri'
-        weekmask of valid business days, passed to ``numpy.busdaycalendar``
+        Weekmask of valid business days, passed to ``numpy.busdaycalendar``.
     holidays : list
-        list/array of dates to exclude from the set of valid business days,
-        passed to ``numpy.busdaycalendar``
+        List/array of dates to exclude from the set of valid business days,
+        passed to ``numpy.busdaycalendar``.
     calendar : pd.HolidayCalendar or np.busdaycalendar
+        A business day calendar.
     offset : timedelta, default timedelta(0)
+        Applies a time offset (from ms to years).
```

Comment:
Still nitpicking but Python time delta only goes up to days so I think this is a misnomer saying years.

Probably better just to say something like `Time offset to apply` and leave any precision questions left to other parts of the documentation

Reply:
Reworded.
I assumed that the offset could be in years because `pandas.Timedelta` can and is said to be interchangeable with it.
Better safe than sorry